kubernetes/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
PET is licensed under the following terms: https://github.com/pytorch/elastic/blob/master/LICENSE. Could we please keep this work consistent with the rest of PET and PyTorch?

Reply:
Yeah, I think we should keep it same. I will update the boilerplate

Reply:
change to following. 

```
/*
Copyright (c) Facebook, Inc. and its affiliates.
All rights reserved.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.
*/

```